pkg/kubelet/kubelet.go

```diff
@@ -837,7 +837,15 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		util.SetNodeOwnerFunc(klet.heartbeatClient, string(klet.nodeName)))
 
 	// setup node shutdown manager
-	shutdownManager, shutdownAdmitHandler := nodeshutdown.NewManager(klet.GetActivePods, killPodNow(klet.podWorkers, kubeDeps.Recorder), klet.syncNodeStatus, kubeCfg.ShutdownGracePeriod.Duration, kubeCfg.ShutdownGracePeriodCriticalPods.Duration)
+	klog.InfoS("Creating node shutdown manager", "shutdownGracePeriodRequested", kubeCfg.ShutdownGracePeriod.Duration, "shutdownGracePeriodCriticalPods", kubeCfg.ShutdownGracePeriodCriticalPods.Duration, "podPriorityShutdownGracePeriods", kubeCfg.PodPriorityShutdownGracePeriods)
```

Comment:
IMO, I think we should move this inside the shutdown manager and only log if feature gate enabled + shutdown manager active. Otherwise, this will be log spam if if shutdown manager is disabled or on other unsupported platforms (e.g. windows).